xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/ClassEditSheet.xml

```diff
@@ -256,7 +256,10 @@ xcontext.put('propertyCustomDisplayer', new PropertyCustomDisplayer(xcontext))
     #set ($action = 'create')
     #set ($discard = $params.put('parent', $doc.fullName))
   #end
-  &lt;span class="$class"&gt;&lt;a href="$escapetool.xml($xwiki.getURL($reference, $action, $escapetool.url($params)))"
+  &lt;span class="$class"&gt;
+  &lt;a href="$escapetool.xml($xwiki.getURL($reference, $action, $escapetool.url($params)))"
```

Comment:
The span and anchor were put on the same line most probably to avoid white space. I don't see any reason to change this line, as it fits the 120 characters limit.